vergedb_test/routes/drones.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const router = express.Router();
+const drones = require('../services/drones');
+
+/* GET drones listing. */
+router.get('/', function (req, res, next) {
+    try {
+        //Commands here
+        res.json(drones.getMultiple(req.query.page));
```

Comment:
rename `getMultiple` to `getAllDrones`.

- [ ] return a list of all drones with no limit